packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -99,9 +99,9 @@ export const RichTextBlock = memo(function RichTextBlock({
 
         <ToolbarDivider />
 
-        <EditorSubMenuButton icon="smile-o" editorHasFocus={hasFocus}>
-          <EmojiPicker doWithEmoji={emoji => editor.insertText(emoji)} />
-        </EditorSubMenuButton>
+        <SubMenuButton icon="smile-o">
```

Comment:
Only renamed the SubmenuButton. No functional changes.

Reply:
👍🏾